setup.py

```diff
@@ -17,17 +17,17 @@
 
 PYTHON_REQUIRES = ">=3.6.0"
 INSTALL_REQUIRES = [
-    "antspyx==0.2.7",
+    "antspyx==0.2.9",
     "imgaug<0.3",
     "matplotlib>=3.0.3",
     "mlflow",
     "nibabel>=2.4.0",
     "numpy",
     "seaborn",
-    "scikit-image>=0.16.0",
+    "scikit-image>=0.17.1",
     "scikit-learn>=0.20.2",
     "scipy",
-    "tensorflow>=2.4.0",
+    "tensorflow>=2.6.0",
```

Comment:
Just curious - the version changes in `antspyx` and `tensorflow` are required by the version change in `scikit-image`?
Maybe we can also leave some comments in the code documenting why version constraints are necessary

Reply:
It seems `antspyx==0.2.7` does not exist anymore. 
```
$ pip index versions antspyx
Available versions: 0.3.1, 0.2.9, 0.2.4, 0.2.2
  INSTALLED: 0.3.1
  LATEST:    0.3.1
```

Reply:
Which python version did you use? With py37 I get
```
Available versions: 0.3.1, 0.2.9, 0.2.7, 0.2.4, 0.2.2
```

Reply:
I don't understand why I have this result locally! But I tried to revert this change of version and the CI is passing now.